### PR TITLE
Rebranding of secplug to Scancloud APIs

### PR DESCRIPTION
**Pull request by Ajaatsecplugs**
Rebranding of secplug to Scancloud APIs

---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by Amjed-Ayoub** [comment]
Hi @Ajaatsecplugs, thank you so much for submitting your pull request. This is a new verified connector. Like all verified publisher connectors, we ask that you register your connector for certification by filling out our [registration form](https://aka.ms/ConnectorRegistration).
Please let me know if you have any questions.
Thank you very much again for working with us.

**Comment by Amjed-Ayoub** [comment]
Apologies for this mistake. You are correct. No need for registration again :-) we will review this pull request soon.
Thank you very much.

File: certified-connectors/secplugs/apiDefinition.swagger.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
   "swagger": "2.0",
   "info": {
-    "title": "Secplugs Scan",
-    "description": "Secplugs Scan for scanning Emails content, files contents and Urls, Secplugs scan is a platform which creates hybrid security platform using various security provider",
+    "title": "Scancloud APIs",
```
Comment (v-EgorKozhadei) [inline certified-connectors/secplugs/apiDefinition.swagger.json:4]: @Ajaatsecplugs please remove the workdAPI from the title and from the description, it's not allowed to use it. You can check the documentation [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#connector-info) and [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#give-your-connector-a-title) for more information.

File: certified-connectors/secplugs/apiDefinition.swagger.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+    "description": "Scancloud APIs for scanning Emails content, files contents and Urls, Scancloud APIs is a platform which creates hybrid security platform using various security provider",
     "version": "1.0",
     "contact": {
-      "name": "Secplugs",
-      "url": "https://secplugs.com/",
+      "name": "Scancloud",
+      "url": "Website	https://scancloud.io/",
```
Comment (v-EgorKozhadei) [inline certified-connectors/secplugs/apiDefinition.swagger.json:9]: @Ajaatsecplugs the URL property should contain only contact URL, please remove the word Website and whitespaces from the string.

**Review by v-EgorKozhadei — COMMENTED** [review]
Hi @Ajaatsecplugs! Please look at the commends after the review, thank you!

File: certified-connectors/secplugs/readme.md [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
-* An account in secplugs
+* An account in Scancloud
 * The Power platform CLI tools
 
 ## Building the connector 
-You would need to ensure that you have an account in secplugs and using https://secplugs.com/index.php/providers an power automate profile is created.
+You would need to ensure that you have an account in secplugs and using https://scancloud.io/index.php/providers an power automate profile is created.
 After that is completed, you can create and test the secplugs connector.
 
 ### Deploying the sample
```
Comment (v-EgorKozhadei) [inline certified-connectors/secplugs/readme.md:20]: @Ajaatsecplugs please format the readme document according to [the readme.md template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md).

File: certified-connectors/secplugs/apiDefinition.swagger.json (hunk 1 above)
Comment (v-EgorKozhadei) [reply certified-connectors/secplugs/apiDefinition.swagger.json:4]: Title update is not allowed for deployed connectors, it's a breaking change. Can you please revert the title to Secplugs Scan? If you want a new name you'll have to submit a new PR following [these guidelines](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission).

**Comment by Ajaatsecplugs** [comment]
We would like to update the Title for the deployed connector - As per your
suggestion, we have followed the guidelines
https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission
and
raised a new PR with the new title.

https://github.com/microsoft/PowerPlatformConnectors/pull/1697

Thanks
Shankar

**Comment by Ajaatsecplugs** [comment]
We have completed the registration and followed the guidelines - Please
find the PR:
https://github.com/microsoft/PowerPlatformConnectors/pull/1697/files

Please review and confirm.

thanks
Shankar

**Comment by Ajaatsecplugs** [comment]
Could you please acknowledge and let us know the ETA to review and merge
the PR?

thanks
Shankar

**Comment by v-EgorKozhadei** [comment]
Hi @Ajaatsecplugs!

We will review this PR within 1-2 weeks.

It turned out that we can't have two connectors with the same functionality under different names.
I'm sorry that we asked you to open a new PR because of the new name.
We return to reviewing this PR, [the new one](https://github.com/microsoft/PowerPlatformConnectors/pull/1697) will be closed soon.

Can you please apply all the file changes that you have made in [the new PR](https://github.com/microsoft/PowerPlatformConnectors/pull/1697) to this PR?

Thank you in advance! And sorry for inconveniences!

File: certified-connectors/Scancloud/apiDefinition.swagger.json [hunk 4]
```diff
@@ -1,12 +1,12 @@
 {
   "swagger": "2.0",
   "info": {
-    "title": "Secplugs Scan",
-    "description": "Secplugs Scan for scanning Emails content, files contents and Urls, Secplugs scan is a platform which creates hybrid security platform using various security provider",
+    "title": "Scancloud",
+    "description": "Scancloud for scanning Emails content, files contents and Urls, Scancloud is a platform which creates hybrid security platform using various security provider",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Scancloud/apiDefinition.swagger.json:5]: @Ajaatsecplugs can you please describe concisely the main purpose and value offered by your connector in the description? You can refer to [the documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector).

File: certified-connectors/Scancloud/apiDefinition.swagger.json [hunk 5]
```diff
@@ -338,7 +338,7 @@
             }
           }
         },
-        "summary": "Scan Emails using SecPlugs Scan",
+        "summary": "Scan Emails using Scancloud",
         "operationId": "emailscan",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Scancloud/apiDefinition.swagger.json:342]: @Ajaatsecplugs please [use pascal casing for operationId](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#actions-and-triggers) properties.

File: certified-connectors/Scancloud/apiDefinition.swagger.json [hunk 6]
```diff
@@ -154,8 +154,8 @@
             "type": "string",
             "default": "",
             "required": false,
-            "description": "Secplugs API Key",
-            "x-ms-summary": "Secplugs API Key"
+            "description": "Scancloud API Key",
+            "x-ms-summary": "Scancloud API Key"
```
Comment (v-EgorKozhadei) [inline certified-connectors/Scancloud/apiDefinition.swagger.json:158]: @Ajaatsecplugs please don't copy paste summaries and descriptions, they should differ. Can you please make a bit more detailed descriptions? You can refer to [English strings guidelines](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).

File: certified-connectors/Scancloud/apiDefinition.swagger.json [hunk 7]
```diff
@@ -347,8 +347,8 @@
             "type": "string",
             "default": "",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Scancloud/apiDefinition.swagger.json:348]: @Ajaatsecplugs empty properties are not allowed. You can remove it until required or define a value. Please refer to [the documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-exact-operation-responses).

File: certified-connectors/Scancloud/readme.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
+
+## Scancloud Connector
```
Comment (v-EgorKozhadei) [inline certified-connectors/Scancloud/readme.md:2]: @Ajaatsecplugs please remove the word Connector from the title.

File: certified-connectors/Scancloud/readme.md [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+Scancloud provides a powerful platform for pluggable security using Scancloud plugins.
```
Comment (v-EgorKozhadei) [inline certified-connectors/Scancloud/readme.md:3]: @Ajaatsecplugs please provide some information about the main purpose and value offered by your connector.

File: certified-connectors/Scancloud/readme.md [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+* `Scan a file`: scans a file, using the configured security provider in scancloud profile.
+* `Scan an email`: scans a file, using the configured security provider in scancloud profile.
+
+## Obtaining Credentials
+You can obtain key using a valid account from ScanCloud Profile.
+
+## Known Issues and Limitations
+N/A
+
+### Deploying the sample
```
Comment (v-EgorKozhadei) [inline certified-connectors/Scancloud/readme.md:25]: @Ajaatsecplugs please format the readme.md file according to [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md?plain=1). All of the required headings should be present.

**Review by v-EgorKozhadei — COMMENTED** [review]
Hi @Ajaatsecplugs! Please check comments after the review, thank you!

**Comment by v-EgorKozhadei** [comment]
Hi @Ajaatsecplugs! Can you please also leave the connector folder name as it is? So the files will be updated, but the connector folder name will stay the same.

File: certified-connectors/secplugs/readme.md (hunk 3 above)
Comment (Ajaatsecplugs) [reply certified-connectors/secplugs/readme.md]: Fixed

File: certified-connectors/Scancloud/readme.md (hunk 10 above)
Comment (Ajaatsecplugs) [reply certified-connectors/Scancloud/readme.md]: Added

File: certified-connectors/Scancloud/readme.md (hunk 9 above)
Comment (Ajaatsecplugs) [reply certified-connectors/Scancloud/readme.md]: Added

**Comment by Ajaatsecplugs** [comment]
Thank you for the review comments - Fixed all the review comments and
raised the PR again..

Thanks
Shankar

**Review by v-EgorKozhadei — APPROVED** [review]

**Comment by v-EgorKozhadei** [comment]
Hi @Ajaatsecplugs!

Your PR is approved and merged. Please submit an update in ISV Studio by using your latest commit ID.

Please let me know if you have any questions.

**Comment by Ajaatsecplugs** [comment]
Powerplatform team,

I understand that scancloud connectors are now live in power
automate connectors site.. I understand that you wanted to treat
Scancloud.io connectors as new connectors not merging with existing
secplugs connectors.. There seems to be some issue here..

1. Secplugs and scancloud connector documentation got mixed up..
https://docs.microsoft.com/en-us/connectors/secplugscan/
This is quite confusing - We would like to stick on to the latest
documentation on scancloud connectors - you can remove the secplugs
documentation all together if you want..

2. Scancloud connectors - says "scan newly added files using Secplugs"..
https://powerautomate.microsoft.com/en-us/connectors/details/shared_secplugscan/scancloud/
As we mentioned earlier we want to replace secplugs completely with
Scancloud.io - this is a rebranding exercise as mentioned earlier.. Please
remove secplugs and replace them with Scancloud..  It should not get mixed
up..

Please let us know if you have any questions.

thanks
Shankar

**Comment by Ajaatsecplugs** [comment]
Hello Shankar,

Thank you so much for reaching out.

You new update deployment did not start yet 😊. Hopefully, our engineers will prepare your update for Friday’s deployment train. Typically, it takes up to 3 to 4 weeks to complete.

Please let me know if you have any questions.

Thank you so much for your patience and understanding.

Best regards,
Amjed Ayoub

From: Shankar Jayaraman ***@***.***>
Sent: Sunday, August 7, 2022 9:07 PM
To: microsoft/PowerPlatformConnectors ***@***.***>
Cc: microsoft/PowerPlatformConnectors ***@***.***>; Kartik Rao Polepalli ***@***.***>; Connector Partner Mgmt Team ***@***.***>; Srikanth Yenagandhula (MINDTREE LIMITED) ***@***.***>
Subject: [EXTERNAL] Re: [microsoft/PowerPlatformConnectors] Rebranding of secplug to Scancloud APIs (PR #1668)

Powerplatform team,

I understand that scancloud connectors are now live in power automate connectors site.. I understand that you wanted to treat Scancloud.io connectors as new connectors not merging with existing secplugs connectors.. There seems to be some issue here..

1. Secplugs and scancloud connector documentation got mixed up..
https://docs.microsoft.com/en-us/connectors/secplugscan/<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdocs.microsoft.com%2Fen-us%2Fconnectors%2Fsecplugscan%2F&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7C261bdc06e82f48f4171d08da78f36f55%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637955285146033130%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C2000%7C%7C%7C&sdata=mjws3CF0KbU%2F%2BjUTP2wf8yXdcBLJga0K8xl1kstqrzo%3D&reserved=0>
This is quite confusing - We would like to stick on to the latest documentation on scancloud connectors - you can remove the secplugs documentation all together if you want..

2. Scancloud connectors - says "scan newly added files using Secplugs"..
https://powerautomate.microsoft.com/en-us/connectors/details/shared_secplugscan/scancloud/<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fpowerautomate.microsoft.com%2Fen-us%2Fconnectors%2Fdetails%2Fshared_secplugscan%2Fscancloud%2F&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7C261bdc06e82f48f4171d08da78f36f55%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637955285146033130%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C2000%7C%7C%7C&sdata=xRUxiLogvbWV7rzoUKlQrc6rh9c%2FClzoPVLsIp8njso%3D&reserved=0>
As we mentioned earlier we want to replace secplugs completely with Scancloud.io - this is a rebranding exercise as mentioned earlier.. Please remove secplugs and replace them with Scancloud..  It should not get mixed up..

Please let us know if you have any questions.

thanks
Shankar

On Sat, 25 Jun 2022 at 02:41, v-EgorKozhadei ***@***.******@***.***>> wrote:

Hi @Ajaatsecplugs<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAjaatsecplugs&data=05%7C01%7Cv-amjedayoub%40microsoft.com%7C261bdc06e82f48f4171d08da78f36f55%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637955285146033130%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C2000%7C%7C%7C&sdata=GumyA%2B1yrKgYkmw5sEtoePy6OJfc5zQRQTSXfAujzG8%3D&reserved=0>!

Your PR is approved and merged. Please submit an update in ISV Studio by using your latest commit ID.

Please let me know if you have any questions.

**Comment by Ajaatsecplugs** [comment]
Okay thanks for the update..  The below message we see in the activity
center..  Hope this is the correct status..

"
Dear Partner,

Your connector is now onboarded to the current deployment schedule. You can
expect another status update in 3-5 business days.

Best Regards, Srikanth Y.
"
Thanks
Shankar

**Comment by Ajaatsecplugs** [comment]
Hi Team - There are some issues with scancloud connector preview  - I can
see our scancloud rebranded connector updated as a preview
https://powerautomate.microsoft.com/en-us/connectors/details/shared_secplugscan/scancloud/
 - Based on our earlier conversation, you have mentioned that you will be
uploading this as a brand new scancloud connector, not an update to the
existing secplugs connector. Also, we have the following concerns. Request
to please address the same..

1. Documentation link still shows secplugsscan
https://docs.microsoft.com/en-us/connectors/secplugscan/

2. In the connector page
https://powerautomate.microsoft.com/en-us/connectors/details/shared_secplugscan/scancloud/
It is showing as secplugs connector - In the URL, connector name etc.,

3. In the documentation "Getting started with your connector" section,
there are some broken links and images, we have provided in our
documentation.. It was there in our earlier secplugs documentation page and
we want to remove that. It looks bad and needs to be removed.. We want to
keep only one line in that section which is " Please follow ScanCloud
Connector documentation here
<https://scancloud.io/docs?plugin=plugin-secplugs-plugins-power-automate>".

Please remove the below section:
"
Creating a connection | DEFAULT | | |
|---------------------------------------------|---------------------|---------------------------------|
| Name | Type | Description | | API Key (from https://Scancloud.com/
<https://scancloud.com/>) | securestring | The Api Key for this API |

Clean report Sample (For File): [image: enter image description here]

Malicious report Sample (For File): [image: enter image description here]
"
Thanks
Shankar

**Comment by Ajaatsecplugs** [comment]
Hello Shankar,

Thank you so much for reaching out.

Please see my answers inline below.

Please let me know if you have any questions.

Thank you so much again for your hard work with us.

Best regards,
Amjed Ayoub

**Comment by Ajaatsecplugs** [comment]
Thanks for the quick response..

We have now updated the intro.md in ISV studio - Could you please take the
updated intro.md file please..

Already regarding the connector failure, we were having some temporary
issues with our APIs and it was down, that could be reason for
the failure.. it is all fixed and it's been up and running for the last few
days without any issues. Could you please check now.

Thanks
Shankar

**Pull request merged**